pages/open-source.tsx

```diff
@@ -119,7 +121,7 @@ const bounties = [
       <>
         Gas optimization is a critical aspect of developing efficient and
         cost-effective smart contracts on blockchain platforms. This bounty aims
-        to support optimizing gas usage for thirdweb&apos;s published smart
```

Comment:
this change breaks the build. `&apos;` is the unicode equivalent for `'` which is why it is here